app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -11,11 +11,14 @@ def initialize
         @validate_resource_restrictions_class = Authentication::ResourceRestrictions::ValidateResourceRestrictions
       end
 
-      def create_authentication_parameters(authenticator_input)
-        @authentication_parameters = @authentication_parameters_class.new(authenticator_input)
-        @authentication_parameters.jwt_token = @extract_token_from_credentials.call(
-          credentials: authenticator_input.request.read
+      def authentication_parameters(authenticator_input)
+        jwt_token = @extract_token_from_credentials.call(
+          credentials: authenticator_input.request.body.read
         )
+        authentication_parameters = @authentication_parameters_class.new(authentication_input: authenticator_input,
+                                                                         jwt_token: jwt_token
+        )
+        authentication_parameters
```

Comment:
how about:
```
```suggestion
        @authentication_parameters_class.new(
          authentication_input: authenticator_input,
          jwt_token: jwt_token
        )
```